### ISSUE TITLE
NVDA not announcing the Menuitemradio checked state when usign MS Edge

### ISSUE THREAD

**Issue by GavinOgstonTR** [nvaccess/nvda#18173]
### Steps to reproduce:

1. Using MS Edge browser
2. Open NVDA with Speech Viewer.
3. Go to the ARIA Authoring Practices menubar example https://www.w3.org/WAI/ARIA/apg/example-index/menubar/menubar-editor.html
4. Tab down to the Example and open the Font menu that uses Menuitemradio for its options.
5. Navigate the options and notice that the selected options does not announce its checked state set with aria-checked="true"

### Actual behavior:

Menu Menuitemradio options are announced, but the selected/checked menu item state is not announced. When using other browsers (Chrome/Firefox), it is announced.

### Expected behavior:

Selected/checked menuitemradio items convey the checked state and are announced by the screen reader.

**Comment by SaschaCowley** [comment]
@GavinOgstonTR can you also reproduce this in Chrome?
What is the setting "Use UIA with Microsoft Edge and other Chromium based browsers when available" in NVDA's advanced settings set to?

**Comment by SaschaCowley** [comment]
Possibly related to #14550

**Comment by GavinOgstonTR** [comment]
@SaschaCowley - This only applies to Edge - Chrome and Firefox are fine.

I'm getting the same result whether I have "Use UIA with Microsoft Edge and other Chromium based browsers when available"  turned on or off.

**Comment by SaschaCowley** [comment]
Thanks for the extra info, @GavinOgstonTR.
